tools/depends/native/config.site.native.in

```diff
@@ -24,3 +24,12 @@ fi
 
 LD_LIBRARY_PATH=@prefix@/@tool_dir@/lib:$LD_LIBRARY_PATH
 NASM=@prefix@/@tool_dir@/bin/yasm
+
+if test "@platform_os@" = "osx" ; then
+  # Xcode 8 + 10.11, clock_gettime and getentropy is present
+  # in 10.12 but will get wrongly detected if building on 10.11
+  ac_cv_search_clock_gettime=no
```

Comment:
btw this very PR seems to compile fine with xcode8 and ios 10 sdk (deployment target bumped to 6.0) ... not runtime tested yet though